src/main/java/com/demoqa/pages/LoginPage.java

```diff
@@ -0,0 +1,37 @@
+package com.demoqa.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+import static org.openqa.selenium.By.cssSelector;
+
+public class LoginPage {
+    WebDriver driver;
```

Comment:
format your code split WebDriver field from constants with empty line. Constants should be set before the field of the class.